playwright/src/fixtures/toc.ts

```diff
@@ -33,7 +33,7 @@ class TOC {
 
   async unitIntroCount() {
     // Total number of unit introduction pages in the book
-    const unitIntroPageLocator = this.page.locator('//li[@data-type="unit"]/details/ol[1]/li[1][@data-type="page"]')
+    const unitIntroPageLocator = this.page.locator('//li[@data-type="unit"]/a/ol[1]/li[1][@data-type="page"]')
```

Comment:
isn't this no longer true? if its true i would be worried about putting lists inside an anchor, but i'm pretty sure you updated that already